eclipse-theia/theia-cpp-extensions
# Remove 'cortex-debug' extension

#### What it does


Fixes #61

- removed the `cortex-debug` extension.
  - the `cortex-debug` extension is removed since it is no longer being actively maintained and there is better cortex debugging support through the VS Code extension alternative.
- removed references of the `cortex-debug` extension from the README.
- removed references of the `cortex-debug` extension from the example
`browser` and `electron` applications.

#### How to test


- Verify that the build passes successfully.
- Verify that the example workspaces still works correctly (tasks, debugging...).

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: vince-fugnitto <vincent.fugnitto@ericsson.com>
